src/api/common/schemas/balance-sheet-options.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "balance-sheet-options",
+  "description": "Options for getBalanceSheet",
+  "type": "object",
+  "properties": {
+    "hotwallet": {
+      "anyOf": [
+        {"$ref": "address"},
+        {"type": "array", "items": {"$ref": "address"}}
+      ]
+    },
+    "strict": {"type": "boolean"},
```

Comment:
Docs say "If true, only accept an address or public key for the account parameter. Defaults to false." What other options are there?